docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -117,6 +117,141 @@ The values represent the speed and direction to command the motor to, normalized
 Refer to the `uavcan.equipment.esc.RawCommand section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rawcommand>`_ in the specification 
 for more details
 
+uavcan.tunnel.Broadcast (Data Type ID = 2010)
+#############################################################
+DroneCAN's “tunnel” messages allow users to transmit arbitrary data bytes through the DroneCAN protocol. The tunnel broadcast message contains a byte representing the 
+protocol being tunneled, a channel ID allowing for additional routing options, and an array of up to 60 bytes of data.
+
+Vertiq modules are configured to accept broadcast messages with a protocol value of 0x55, and check the received data buffer for valid :ref:`IQUART data <uart_messaging>`. 
+As such, the DroneCAN tunnel broadcast message allows users to transmit IQUART messages via DroneCAN.
+
+A very rudimentary example of using the DroneCAN tunnel message is provided below. The example requires an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_, and two :ref:`FTDI devices <connection_guide>`. 
+The hardware is configured as follows:
+
+.. image:: ../_static/manual_images/dronecan/dronecan_tunnel_hardware.png
+
+In this example, the module has no serial connection to :ref:`IQ Control Center <control_center_start_guide>`. Instead, the Control Center sends messages to a Python script which packages 
+the IQUART bytes as a DroneCAN tunnel message, and transmits the message to the module. The module receives and decodes the IQUART message, and if necessary, 
+forms a response as a DroneCAN tunnel message which is sent back to the Python script. Finally, the Python script pulls the IQUART response out of the module's 
+tunnel message, and transmits the response to the Control Center via IQUART. The process is summarized below.
+
+.. image:: ../_static/manual_images/dronecan/dronecan_tunnel_python_sequence.png
+
+The Python script necessary to accomplish this is provided here: 
+
+.. code-block:: python
+
+	import time
+	import iqmotion as iq
+	import pyuavcan_v0 as uavcan
```

Comment:
Any chance you could see if we can use pydronecan instead? This is an old library, and I feel like we will get laughed at by the all cool dronecan kids if they see it: https://github.com/dronecan/pydronecan